app/views/hub/sites/_form.html.erb

```diff
@@ -7,6 +6,21 @@
     <%= f.cfa_input_field :name, t("general.name") %>
     <%= f.cfa_select :parent_organization_id, t("general.organization"), @organizations.map { |c| [c.name, c.id] } %>
 
+    <% if @show_unique_links %>
+      <% @site.source_parameters.each do |source_parameter| %>
+        <%= f.fields_for :source_parameters, source_parameter, builder: VitaMinFormBuilder do |source_parameter_form| %>
+          <%= source_parameter_form.cfa_input_field :code, t("general.unique_link"), classes: ["form-width--short"] %>
+          <%= source_parameter_form.cfa_checkbox(:_destroy, t("general.remove")) %>
```

Comment:
Consider making this checkbox be in the same "row" visually as the form input it relates to.

Reply:
If there's a way to save the deletion on click of the 'Remove', that might be a more common pattern.